### PR TITLE
add request model

### PR DESCRIPTION
**Pull request by JunZheng007**
### What this PR does (required):
add following properties to request model
- user_id (user who is requesting)
- sitter_id (pet sitter user id)
- start (date requested to start)
- end (date requested to end)
- accepted (default False)
- declined (default False)
- paid (default False)

### CODE REVIEW

File: server/models/Request.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  declined: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
+  paid: {
+    type: Boolean,
+    required: true,
+    default: false,
+  },
```
Comment (ezeilo-su) [inline server/models/Request.js:36]: You can also add the mongoose `timestamps` option to tell Mongoose to automatically manage createdAt and updatedAt properties on the document.

**Review by ezeilo-su — COMMENTED** [review]
Schema looks good!

File: server/models/Request.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  endDate: {
+    type: Date,
+    required: true,
+  },
+  accepted: {
```
Comment (rajivtitus) [inline server/models/Request.js:22]: We can have an enum type object to keep track of the request status

```   status: {
        type: String,
        enum : ['pending','accepted', 'declined'],
        default: 'pending'
        lowercase: true,
    },
```

**Comment by rajivtitus** [comment]
@JunZheng007 Please wait for PR approvals before merging :)

File: server/models/Request.js (hunk 2 above)
Comment (JunZheng007) [reply server/models/Request.js:22]: fix it on PR #59

**Pull request merged**